comment/views.py

```diff
@@ -0,0 +1,27 @@
+from .models import BlogPost
+from .forms import CommentForm
+from django.shortcuts import render, get_object_or_404
+# Create your views here.
+
+class post_detail(request, slug):
```

Comment:
You could decorate this view with:
`@require_http_methods(["POST"])` 

Then you won't need to check if the request method is POST or not.